.github/workflows/test-pr.yml

```diff
@@ -198,7 +198,7 @@ jobs:
       - name: 'Build Foundry'
         run: docker exec -u github-user kevm-ci-foundry-${GITHUB_SHA} /bin/bash -c 'make build-foundry -j2'
       - name: 'Foundry Prove'
-        run: docker exec -u github-user kevm-ci-foundry-${GITHUB_SHA} /bin/bash -c 'make test-foundry-prove PYTEST_ARGS=-vv'
+        run: docker exec -u github-user kevm-ci-foundry-${GITHUB_SHA} /bin/bash -c 'make test-foundry-prove PYTEST_ARGS=-vv --no-use-booster'
```

Comment:
This `--no-use-booster` flag logic should be added somewhere, you could either:
- rename all uses of `use_booster` in pytest tests
- handle the logic in the `Makefile` by writing `--use-booster` if the flag `NO_USE_BOOSTER` is false
As you want to make the booster the default, 1. might be a better choice and a less confusing one

Reply:
Actually, what about writing 
```diff
diff --git a/kevm-pyk/src/tests/conftest.py b/kevm-pyk/src/tests/conftest.py
index 7a2b381d3..58cd4dcd6 100644
--- a/kevm-pyk/src/tests/conftest.py
+++ b/kevm-pyk/src/tests/conftest.py
@@ -30,4 +30,4 @@ def update_expected_output(request: FixtureRequest) -> bool:

 @pytest.fixture(scope='session')
 def use_booster(request: FixtureRequest) -> bool:
-    return request.config.getoption('--use-booster')
+    return not request.config.getoption('--no-use-booster')
```

Reply:
Thanks @iFrostizz for the suggestion, looks very good! Done in https://github.com/runtimeverification/evm-semantics/pull/2044/commits/393e42b820e6c1c6e668220889980383711e8a0b.

Reply:
I agree that it's okay to leave the CI commands as they are @palinatolmach , because they are not user facing, and because we *may* drop non booster CI jobs in the future. Also, the `--use-booster` option is more straightforward, having `--no-use-booster` features a double negation unless you edit further the tests, it looks like unnecessary effort!